packages/core/data-transfer/src/strapi/providers/remote-destination/index.ts

```diff
@@ -143,9 +181,20 @@ class RemoteStrapiDestinationProvider implements IDestinationProvider {
             return callback(streamError);
           }
         }
-        const e = await this.#endStep(step);
+        const { error, stats } = await this.#endStep(step);
+
+        if (
+          stats &&
+          (stats.started !== this.stats[step].count || stats.finished !== this.stats[step].count)
+        ) {
```

Comment:
Just to improve readability

```suggestion
        const { count } = this.stats[step];
        
        if (stats && (stats.started !== count || stats.finished !== count)) {
```